components/card.css

```diff
@@ -0,0 +1,119 @@
+@import url(./button.css);
+.card {
+  background-color: var(--white);
+  border: 1px solid #555;
```

Comment:
change border color #555 into var(--bordercolor). if you are using this color many times .